index.bs

```diff
@@ -223,6 +227,7 @@ interface ImageCapture {
     double          imageHeight;
     double          imageWidth;
     boolean         redEyeReduction;
+    DOMString       imageFormats;
```

Comment:
Same here: s/imageFormats/imageFormat/ to follow the `fillLightMode` convention (and `exposureMode` etc).